modules/lti/src/components/Upload.tsx

```diff
@@ -108,6 +111,42 @@ class TranslatedUpload extends React.Component<UploadProps, UploadState> {
         }));
     }
 
+    componentWillUnmount() {
```

Comment:
You removed this lock in pull request #2263
This probably shouldn't be in here, should it?